autoconnect/autoconnect-web/src/error.rs

```diff
@@ -25,7 +29,11 @@ impl ResponseError for ApiError {
 
     fn error_response(&self) -> HttpResponse {
         let code = self.status_code();
-        HttpResponse::build(code).json(json!({
+        let mut resp = HttpResponseBuilder::new(code);
```

Comment:
We definitely want this in autoendpoint. I'm not sure there's a really compelling reason for it in autoconnect but I suppose it wouldn't hurt to have it here as well for its main websocket route "/".